### PR TITLE
Created the database connection.

### PR DESCRIPTION
**Pull request by moustf**
- Created db environment variables.
- Created .example.env file.
- Created the config directory in which we will export the environment variables.
- Created the connection functionality.

### CODE REVIEW

**Review by Bas-Shiekh — DISMISSED** [review]

**Review by mahmoudhammad309 — APPROVED** [review]

**Review by Aaya-Elsharief — DISMISSED** [review]

File: server/database/connection.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Sequelize } from 'sequelize';
+import envVars from '../config/environment';
+
+const {
+  dbDev, dbTest, dbProduction, nodeEnv,
+} = envVars;
+
```
Comment (Raghad-Mezied) [inline server/database/connection.ts:7]: You can import your variables instead of import envVars

File: server/database/index.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import sequelize from './connection';
+
+export default {
+  sequelize,
+};
```
Comment (Raghad-Mezied) [inline server/database/index.ts:5]: No need for this file

**Review by Raghad-Mezied — CHANGES_REQUESTED** [review]

File: server/database/connection.ts (hunk 1 above)
Comment (moustf) [reply server/database/connection.ts:7]: I tried to destructure them but it didn't work!

Comment (moustf) [reply server/database/connection.ts]: Solved!

**Review by Bas-Shiekh — APPROVED** [review]

**Review by Raghad-Mezied — APPROVED** [review]

**Review by mahmoudhammad309 — APPROVED** [review]

**Pull request merged**